### ISSUE TITLE
Can't push all regular blocks to sub-sites without turning to Classic

### ISSUE THREAD

**Issue by madriverweb** [10up/distributor#746]
Getting ready to push 45 pages internally on a multi to sub-sites (must do today). First several pages are coming through with Classic Block replacing outermost columns. I've read your known issues/caveats and understand that all blocks being pushed need to be available on the sub-sites, and they are. I'll reference both https://github.com/10up/distributor/issues/739 which I wrote and https://github.com/10up/distributor/issues/296 regarding reusable blocks. In the test pages, have converted all reusable to regular blocks. Even tried pushing a page with no reusable blocks, however every page was created initially with a column block template to start. In other words, I had a reusable page template with two columns, at least one of which had another reusable block in it and a CSS class associated with the column, thus the reason for a template.

Is this what is thwarting the push? I don't see any trace of reusable in the source code of the pushed files. What else could be causing this?

I set both the parent and sub-sites to Block editor as default.

**Comment by madriverweb** [comment]
Still on 1.6.2 stable.

**Comment by jeffpaul** [comment]
@madriverweb it'll help to also understand your WordPress version and any active theme and plugins.  Updating to Distributor 1.6.4 will also help with a couple updates in the last two minor releases, but I don't think either did anything to address content being dumped into a Classic block.  I'll look to run some tests locally with blocks inside a column block to see if that does anything odd, but in the meantime sharing more details about your setup and (potentially) update Distributor will help ensure we're best suited to triage for you.

**Comment by madriverweb** [comment]
This is a follow-up comment for anyone experiencing similar issues. I have had offline conversations with Jeff. Not intending to speak for him, but he suspects the outer column block may be a factor. The reason for the outermost column block in every page was to give it a content/sidebar structure. This would seem to me like a sensible and perhaps common way to establish a site with varying sidebars while moving away from the Widgets method and possibly having to create unique sidebars for certain pages which has been a common practice of mine.

After testing the push as noted above in initial post, I found a better procedure. First I created all the necessary reusable blocks in each sub-site. Then pushed the pages without converting any reusable blocks. (This still resulted in a Classic block.) I then copied the source code from the parent site into each sub-site to get the proper Blocks coding and configuration. In sub-site, switch to visual, observe churning of reusable blocks it can't find, then remove/replace with those in that sub-site. Although it was a lot of work copying the source code roughly 150 times, there was still significant benefit in this process with meta descriptions and images pushed to the sub-sites. Without Distributor I would have had to do some version of copy and paste from the old websites anyway.

For the record, I was on WP 5.6 at the time, Genesis by StudioPress now WPEngine, Distributor 1.6.2 at the time.
